pkg/core/helper/errors.go

```diff
@@ -0,0 +1,8 @@
+package helper
```

Comment:
This is not really go-way of making helers, utils or other package names.

All code belongs to one domain should be in the same package.

Reply:
Agreed, don't know why I did this way.
I opened a ticket EES-4269